packages/training/Dockerfile

```
@@ -0,0 +1,75 @@
+#
+# Run (4x GPU tensor parallel):
+#   docker run --gpus all -v $(pwd)/trained_models:/app/trained_models babylon-training \
+#     --profile l40-4gpu --steps 10000
+
+FROM nvidia/cuda:12.1-runtime-ubuntu22.04
+
+# Prevent interactive prompts
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)